helper/colorlogger.py

```diff
@@ -14,7 +18,7 @@ def create_logger(name, color='reset', log_level=logging.INFO, log_colors={
     handler.setFormatter(colorlog.ColoredFormatter('%(log_color)s%(asctime)s %(' + color + ')s[%(module)10s] %(log_color)s[%(levelname)5s] %(' + color + ')s%(message)s',
                                                    log_colors=log_colors))
     log = colorlog.getLogger(name)
-    log.propagate = False
+    log.propagate = True
```

Comment:
if added a nullhandler by default to the root scope. i know it's not ideal. but it's also not ideal to stop propagation, because propagating is the only way to catch all logs within the root scope and add handlers there